src/System.Private.CoreLib/shared/System/Threading/ThreadPool.cs

```diff
@@ -21,16 +21,13 @@
 using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 using Internal.Runtime.CompilerServices;
-using Microsoft.Win32;
+
+using Thread = Internal.Runtime.Augments.RuntimeThread;
 
 namespace System.Threading
 {
     internal static class ThreadPoolGlobals
     {
-        //Per-appDomain quantum (in ms) for which the thread keeps processing
-        //requests in the current domain.
-        public const uint TP_QUANTUM = 30U;
-
         public static readonly int processorCount = Environment.ProcessorCount;
 
         public static volatile bool vmTpInitialized;
```

Comment:
Nit: It may be nice to strip `VM` from the names in the shared code. The threadpool is independent on VM in CoreRT, and I expect it will be the case on all runtimes eventually.

Reply:
Yeah, it was bothering me a bit too. I will do it in the next round of clean-up.